src/bindings/python/docs/test_examples.md

```diff
@@ -43,6 +51,21 @@ To run full test suite one can utilize `tox` command:
 tox
 ```
 
+### Check codestyle of Python API
+For checking the codestyle of Python API execute this command:
+```
+python -m flake8 ./src/openvino --config=setup.cfg
+```
+For checking the codestyle of compatibility nGraph Python API execute this command:
+```
+python -m flake8 ./src/compatibility/ngraph --config=setup.cfg
+```
+For checking the codestyle of compatibility InferenceEngine Python API execute this command:
+```
+cd src/compatibility/openvino
+python -m flake8 ./ --config=setup.cfg
+```
+
```

Comment:
If there's flake8 here, maybe we could also use `mypy`?
`python -m mypy ./ --config-file ./setup.cfg`
`python -m mypy ./src/compatibility/ngraph --config-file ./setup.cfg`
`python -m mypy ./src/openvino --config-file ./setup.cfg`

Reply:
All of it is already done [here](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/docs/contributing.md#always-test-out-our-code-dont-forget-about-it-before-pushing-and-triggering-cis).

You can link both files against each other. I would like to keep codestyle in general contribution guide. Please link test_examples there.

Reply:
added links also to other files.